fe-mock/src/EvidenceTable.js

```diff
@@ -0,0 +1,174 @@
+import { Fragment } from 'react'
+import { Col, Row, } from 'react-bootstrap'
+
+const EvidenceTable = ({ evidence, evidenceRoot, evidenceIgnore, translate }) => {
+    
+    const parser = new DOMParser()
+    const xmlEvidence = parser.parseFromString( evidence, "application/xml")
+    const xmlNS = xmlEvidence.createNSResolver(xmlEvidence)
+    const xmlRoot = xmlEvidence.evaluate(
+        evidenceRoot, 
+        xmlEvidence,
+        xmlNS,
+        XPathResult.ANY_UNORDERED_NODE_TYPE,
+        null).singleNodeValue
+
+    console.log("xmlRoot", xmlRoot)
+
+    const printNode = (node) => {
+        return <Row className='evidenceHeading'>
+            <Col>
+                <h2>{translate(`canonicalEvidenceFields.${node.localName}`)}</h2>
+            </Col>
+        </Row>
+    }
+
+    const printLeaf = (node) => {
```

Comment:
nittygrittypicky...
(leaf) maybe instead of (node)? since the above function is named printNode and this one printLeaf